### ISSUE TITLE
inspection flag must be 'on' or 'off', ignoring for 'match'

### ISSUE THREAD

**Issue by thrix** [rpminspect/rpminspect#329]
@dcantrell not sure where is this problem, but we see this with:

rpminspect-1.4-0.1.202103101514git.fc32.x86_64
rpminspect-data-redhat-1.4-2.fc32eng.noarch

```
$ rpminspect-redhat -v -w tmpxp9OM_/artifacts -o tmpxp9OM_/rpminspect-results.txt -F json -T ALL -p scl go-toolset-1.15-1.15.8-3.el7_9
*** inspection flag must be 'on' or 'off', ignoring for 'match'
*** Unknown inspection: `match`
```

**Comment by dcantrell** [comment]
Investigating

**Comment by dcantrell** [comment]
The problem is the config file parser is hitting the inspections block reader when it's seeing the specname block.  I've seen this on another inspection, so I can fix that up quickly.  Update coming shortly.

**Comment by dcantrell** [comment]
This bug was a fun birthday present (it's my birthday!).  But I have fixed it.

The issue was with block depth handling in my YAML reading loop.  It worked fine with the standard configuration file, but profiles are subsets of the larger config file which is what triggered the reading problem.  After looking at this for some hours I realized I needed a better way to know what level of the file I was at and if at level 1 then reset my block and group indicators.

TL;DR I fixed it.  Patch coming momentarily.

**Comment by thrix** [comment]
@dcantrell thanks for the update, we will get this in production witht the next release in ~week
